data/sovereign/operationData.go

```diff
@@ -0,0 +1,41 @@
+package sovereign
+
+import "math/big"
+
+// Operation holds the data needed for a bridge transfer
+type Operation struct {
```

Comment:
Can you specify that these are for outgoing operations?
Also, when bridging out operations, we need an array of operations, not only one.
Is that being taken care of in node?

Reply:
Comment updated.
Each `Operation` will be serialized, then all serialized operations will be concatenated and sent to `executBridgeOps`